### DISCUSSION TITLE
UI Change and/or Setting Option

### DISCUSSION THREAD

**Discussion by jeremygifford** [NUWCDIVNPT/stig-manager#124] [Ideas and Enhancements]
The multiple in-app tabs can be useful but it would be nice to have a UI setting for them and/or a right-click context menu with the option to open in a new tab - instead of the side-bar objects being opened in new tabs by default.

**Comment by cd-rite** [comment]
Thanks, Jeremy.
We will keep this in mind next time we take a pass over UI features.
We have also considered implementing a "User Preferences" feature where the default behavior for this action might be stored as well.
That sort of feature could also store opened tabs for the next session, etc.

**Reply by silvercoulter** [reply]
Really enjoying the tool! Within user preferences, it would be nice to have a dark mode and some sort of font size selection which would make the site easier to read. The old StigViewer had those things and though cumbersome, they were there and I used them.

**Comment by csmig** [comment]
I vote for emulating the tab management implemented by Visual Studio Code. A single-click on a navigation tree leaf node opens an ephemeral tab (with an italicized title) that is replaced by clicking another leaf node. Double-clicking a leaf node opens a "permanent" tab.

**Reply by cd-rite** [reply]
Created issue #168 to track this enhancement!

**Comment by csmig** [comment]
Addressed by PR #172. Thanks @jeremygifford for the idea. Hopefully this a an acceptable solution that minimizes the number of Tabs when just browsing around.
